### PR TITLE
Arm64 debug issues

### PR DESCRIPTION
**Pull request by sdmaclea**

### CODE REVIEW

File: src/debug/daccess/request.cpp [hunk 1]
```diff
@@ -616,7 +616,7 @@ ClrDataAccess::GetRegisterName(int regNum, unsigned int count, __out_z __inout_e
         W("X7"),
         W("X8"),  W("X9"),  W("X10"), W("X11"), W("X12"), W("X13"), W("X14"), W("X15"), W("X16"), W("X17"), 
         W("X18"), W("X19"), W("X20"), W("X21"), W("X22"), W("X23"), W("X24"), W("X25"), W("X26"), W("X27"), 
-        W("X28"), W("Fp"),  W("Sp"),  W("Lr")
+        W("X28"), W("Fp"),  W("Lr"),  W("Sp")
```
Comment (sdmaclea) [inline src/debug/daccess/request.cpp]: On `arm64` 
- `LR` is register `X30`, 
- `SP` is register `X31`, 

Correct order of names

File: src/debug/inc/dbgipcevents.h [hunk 2]
```diff
@@ -1172,9 +1172,9 @@ struct MSLAYOUT DebuggerREGDISPLAY
     #define DebuggerIPCE_FloatCount 32
 
     SIZE_T  X[29];
-    SIZE_T  SP;
     SIZE_T  FP;
     SIZE_T  LR;
+    SIZE_T  SP;
```
Comment (sdmaclea) [inline src/debug/inc/dbgipcevents.h]: On `arm64` 
- `FP` is register `X29`, 
- `LR` is register `X30`, 
- `SP` is register `X31`, 

Reorder this structure to make it consistent with all other arm64 context structures

File: src/debug/daccess/request.cpp (hunk 1 above)
Comment (hoyosjs) [reply src/debug/daccess/request.cpp]: This is a nice fix to have, although only SOS uses this.

**Comment by MeiChin-Tsai** [comment]
Approved for 3.0 since still in M2 approval.

**Pull request merged**